packages/api/controllers/nomination.js

```diff
@@ -130,7 +130,8 @@ const updateNomination = async (req, res) => {
           });
 
           nomination.update({
-            readyForBoardReviewTimestamp: Date(),
+            // readyForBoardReviewTimestamp: Date(), // this commented code fixes #326 and we're not sure what it
+            // was supposed to do originally, so we should keep it commented and not fully delete it.
```

Comment:
As I understand #326, we *want* the declined nominations to show up in the list, right? This field is necessary for the grant cycle endpoint to return this nomination (https://github.com/the-difference-engine/ksf/blob/develop/packages/api/controllers/grantCycle.js#L103).

What are you trying to accomplish here?

Reply:
From what we could see, and we couldn't test in the local environment because we couldn't generate past data, it looks like that line updates the readyForBoardReviewTimestamp to the present date. If the grant cycle is in the past (today's date is not included in the cycle), then it is no longer in the grant cycle and is excluded from the list.  In the staging app, if you create a grant cycle that contains todays date, any applications declined today will still appear in the cycle. As soon as you move to a grant cycle that doesn't  include today's date, the application will disappear as soon as it's declined.

Reply:
@Geoff7709 confirm with @sowmya20189 and other product people about what exactly they want because line 133 above was meant to satisfy their earlier requirements.